### ISSUE TITLE
BUG 9.0 NodeMove should respect `aggregate: true`

### ISSUE THREAD

**Issue by mhsdesign** [neos/neos-ui#3587]
otherwise with gather all its not possible to move content around if they dont have the same parent in all dimensions: https://github.com/neos/neos-development-collection/issues/5368

Also for content it might just not be desired to be in sync.

see https://github.com/neos/neos-ui/blob/5d69c44b337b77ed344bc100f605a13722e6ecb0/Classes/Domain/Model/Changes/MoveInto.php#L90

```php
$nodeType->isAggregate()
    ? RelationDistributionStrategy::STRATEGY_SCATTER
    : RelationDistributionStrategy::STRATEGY_GATHER_ALL
```

Related to https://github.com/neos/neos-development-collection/issues/3340

basically currenlty everything is aggregate ^^

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

#### Expected behavior

[What you expected to happen]

#### Actual behavior

[What actually happened]

### Affected Versions

Neos:

UI:

You can get this information by running `composer show` or using the package management module within Neos.
The UI version can also be found inside the left sidebar after version 1.0.3.

**Comment by mhsdesign** [comment]
To avoid having to write test for this behaviour in the neos ui we could introduce a `RelationDistributionStrategy::STRATEGY_DEFAULT` enum on cr basis so it could determine the respective strategy by `SCATTER` or  `GATHER_ALL` and it falls back to the node type. Possibly its also better to avoid using `isAggregate` and to introduce a new schema option:

```yaml
My:NodeType:
   relationDistributionStrategy: STRATEGY_GATHER_ALL 
```

**Comment by ahaeslich** [comment]
Possibly could be moved to 9.1

**Comment by mhsdesign** [comment]
Resolved via https://github.com/neos/neos-ui/pull/3876
